src/appengine/handlers/cron/grouper.py

```diff
@@ -171,6 +174,7 @@ def _group_testcases_based_on_variants(testcase_map):
 
     # Determine anomalous candidates.
     threshold = VARIANT_THRESHOLD_PERCENTAGE * project_num_testcases[project]
+    threshold = min(threshold, VARINAT_MAX_THRESHOLD)
```

Comment:
Can we just simplify the below and do:

```
# Clamp threshold. 
threshold = min(threshold, VARINAT_MAX_THRESHOLD)
threshold = max(VARIANT_MIN_THRESHOLD, threshold)
```

Then we can delete that if statement on line 179.